src/astUtils/visitors.ts

```diff
@@ -55,7 +55,7 @@ export function walk<T>(keyParent: T, key: keyof T, visitor: WalkVisitor, option
     }
 
     if (!element.walk) {
-        throw new Error(`${keyParent.constructor.name}["${key}"]${parent ? ` for ${parent.constructor.name}` : ''} does not contain a "walk" method`);
+        throw new Error(`${keyParent.constructor.name}["${String(key)}"]${parent ? ` for ${parent.constructor.name}` : ''} does not contain a "walk" method`);
```

Comment:
I was getting this error:
`Implicit conversion of a 'symbol' to a 'string' will fail at runtime. Consider wrapping this expression in 'String(...)'.ts(2731)`